packages/digitalocean-management/infrastructure/kustomization.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../../modules/digitalocean/provider
+  - ../../../modules/digitalocean/project
```

Comment:
To unify the setup for now (like other providers), let's assume the project is created.
So let's remove the `project` object from the package and only keep it in the module.